### ISSUE TITLE
clyde prefers local .install files over files in aur even when the packages are different

### ISSUE THREAD

**Issue by anish** [Kiwi/clyde#116]
clyde is being invoked in a directory that already has a "firefox.install" file used for an abs build. When I try to install any other aur package, it only sees the local firefox.install file. 

```
anish@cerveza firefox > sudo clyde -S google-chrome-dev
:: google-chrome-dev package not found, searching for group...
:: google-chrome-dev group not found, searching AUR...

==> Installing the following packages from AUR                                                                                               
Targets (1): google-chrome-dev  

==> Proceed with installation? [Y/n] y
==> Downloading google-chrome-dev...
==> Customizing google-chrome-dev...
==> WARNING: Packages from the AUR are potentially dangerous!
Edit the PKGBUILD? (recommended) [Y/n] n
Edit the firefox.install? (recommended) [Y/n]                             <-- wrong file name

anish@cerveza firefox > sudo clyde -S libjpeg-turbo-svn
:: libjpeg-turbo-svn package not found, searching for group...
:: libjpeg-turbo-svn group not found, searching AUR...

==> Installing the following packages from AUR                                                                                               
Targets (1): libjpeg-turbo-svn  

==> Proceed with installation? [Y/n] y
==> Downloading libjpeg-turbo-svn...
==> Customizing libjpeg-turbo-svn...
==> WARNING: Packages from the AUR are potentially dangerous!
Edit the PKGBUILD? (recommended) [Y/n] n
Edit the firefox.install? (recommended) [Y/n]                             <-- wrong file name
```

**Comment by anish** [comment]
Just to be clear, it is only asking if I want to edit "firefox.install" instead of "google-chrome-dev.install" or "libjpeg-turbo-svn.install"

**Comment by anish** [comment]
accidentally closed this, please re-open

**Comment by nogweii** [comment]
Re-opened, edited formatting.

**Comment by juster** [comment]
Chdir into source pkgdir before customizing. Closed by 211e8a29fa7a97eb7f50219b3ece22db39f60f67.

**Comment by juster** [comment]
This should be fixed when the new pacman3.5 version is released any day now.
